graylog2-web-interface/src/components/common/EntityDataTable/contexts/SelectedEntitiesProvider.tsx

```diff
@@ -25,18 +25,22 @@ import type { EntityBase } from '../types';
 
 type Props<Entity extends EntityBase> = React.PropsWithChildren<{
   initialSelection?: Array<string>,
-  onChangeSelection: (selectedEntities: Array<Entity['id']>) => void
+  onChangeSelection: (selectedEntities: Array<Entity['id']>, data: Readonly<Array<Entity>>) => void,
```

Comment:
We can add another context (we can call it DataContext or EntityDataTableContext) and place there data table entities. Then will be know needs to add data as a func parameter. Also will be no needs to pass entities as a props. 

Later we can also add there other data to this  EntityDataTableContext avoid props drilling f.e. columns, columnRenderersByAttribute etc

Reply:
I like the idea of creating a context in `EntityDataTable` which contains the entities, but in this case we need the data in `IndexSetFieldTypeList:86`, which will not be a child of the context.